src/ardrive.ts

```diff
@@ -217,29 +217,29 @@ export class ArDrive extends ArDriveAnonymous {
 	async movePublicFile(fileId: FileID, newParentFolderId: FolderID): Promise<ArFSResult> {
 		const driveId = await this.getDriveIdAndAssertDrive(newParentFolderId);
 
-		const originalFileMetaData = await this.getPublicFile(fileId);
```

Comment:
Were all these naming changes in the move file functions necessary in this PR? It seems like you may have been going down a path to unify file and/or folder moving and ended up going with separate functions...